define/fsd.md

```diff
@@ -188,6 +188,71 @@ When reading JSON, conforming clients and servers may match property names case-
 
 However, when writing JSON, conforming clients and servers must always use [standard JSON](https://www.json.org/) with no comments, correctly-cased property names, correctly-typed property values, etc.
 
+#### Validation
+
+Validation requirements can be specified with `[validate]`. This attribute results in invalid requests if decorated fields are unspecified in request parameters.
```

Comment:
> This attribute results in invalid requests if decorated fields are unspecified in request parameters.

This seems to imply that all validated fields are required, but that is not (or should not) be the case.

Reply:
Whoops. Copy pasta.